base/util/url.go

```diff
@@ -53,10 +65,10 @@ func MakeDataURL(contents []byte, currentCompression *string, allowCompression b
 			return
 		}
 		gz := ";base64," + base64.StdEncoding.EncodeToString(buf.Bytes())
-		// Account for space needed by "compression": "gzip".
-		if len(gz)+25 < len(opaque) {
+		// Account for space needed by the "gzip" compression value
+		if len(gz)+4 < len(opaque) {
```

Comment:
I would make this `len(gz)+len("gzip")` or so...the compiler should constant fold hopefully, and it makes it more robust to future changes.